server/index.js

```diff
@@ -1,20 +1,32 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-// const mongoose = require('mongoose');
 const { Tent, Shirt, db } = require('../db/index.js');
-// const { tentData, shirtData } = require('../db/seed.js');
 let app = express();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(__dirname + '/../client/dist'));
 
+app.get('/data/shirts', (req, res) => {
+  Shirt.aggregate([ { $sample: { size: 6 } } ]).exec((err, data) => {
+    if (err) {
```

Comment:
Can shorten with es6: if (err) { console.log('Server Error', err); } 
                                       res.status(200).send(data);

Reply:
I'm not sure that is a shorthand for es6 if/else statements. I quickly tried looking up documentation on it and couldn't find anything. If you find something let me know because that will be a nice nugget of knowledge.